src/Uno.UI/UI/Xaml/Controls/ItemsControl/ItemsControl.cs

```diff
@@ -1115,9 +1106,43 @@ internal DependencyObject GetContainerForIndex(int index)
 		{
 			var item = ItemFromIndex(index);
 
-			var container = IsItemItsOwnContainerOverride(item)
-				? item as DependencyObject
-				: GetContainerForItemOverride();
+			var isItsOwnItemContainer = IsItemItsOwnContainerOverride(item);
```

Comment:
Codacy found an issue: [Remove the unused local variable 'isItsOwnItemContainer'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5961910)